Yoast/wordpress-seo
# Change close wizard to link

## Context


* Changes the configuration wizard to match the latest design

## Summary


This PR can be summarized in the following changelog entry:

* Changes the configuration wizard to match the latest design

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and the [corresponding PR on the monorepo](https://github.com/Yoast/javascript/pull/633).
* Open the configuration wizard
* Check the different steps to see if they match the new design

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-24
